sign_language_datasets/datasets/dgs_corpus/dgs_corpus.py

```diff
@@ -129,6 +133,22 @@ def load_split(split_name: str) -> Dict[str, List[str]]:
     return split
 
 
+DEFAULT_FPS = 50
+
+
+class DgsCorpusConfig(SignDatasetConfig):
+    def __init__(self, data_type: Literal['document', 'sentence'] = 'document', split: str = "3.0.0-uzh-document",
```

Comment:
Here I can see that users could use incompatible arguments. Possible solutions:
- for known splits, store whether they are sentence level or document level, and check
- use as a convention that the end of the split identifier must be "document" or "sentence", and this must correspond to `data_type`